packages/netsuite-adapter/test/adapter.test.ts

```diff
@@ -198,11 +202,155 @@ describe('Adapter', () => {
 
       expect(suiteAppImportFileCabinetMock).not.toHaveBeenCalled()
     })
+    // TODO: UT -> fetch include / exclude = undefined
+    // all formats together
+    // only old formats
+    describe('fetchConfig', () => {
+      const configWithoutFetch = { ...omit(config, FETCH) }
+      it('without fetch config, should fetch all types and instances', async () => {
+        const adapter = new NetsuiteAdapter({
+          client: new NetsuiteClient(client),
+          elementsSource: buildElementsSourceFromElements([]),
+          filtersCreators: [firstDummyFilter, secondDummyFilter],
+          config: configWithoutFetch,
+          getElemIdFunc: mockGetElemIdFunc,
+        })
+        const { elements, isPartial } = await adapter.fetch(mockFetchOpts)
+        expect(isPartial).toBeFalsy()
+        expect(elements).toHaveLength(getMetadataTypes().length)
+        const customObjectsQuery = (client.getCustomObjects as jest.Mock).mock.calls[0][1]
+        expect(customObjectsQuery.isTypeMatch('any kind of type')).toBeTruthy()
+        const fileCabinetQuery = (client.importFileCabinetContent as jest.Mock).mock.calls[0][0]
+        expect(fileCabinetQuery.isFileMatch('any/kind/of/path')).toBeTruthy()
+      })
+      it('with defined fetch with no values, should fetch all types and instances', async () => {
+        const configWithEmptyDefinedFetch = { ...configWithoutFetch, [FETCH]: {} }
+        const adapter = new NetsuiteAdapter({
+          client: new NetsuiteClient(client),
+          elementsSource: buildElementsSourceFromElements([]),
+          filtersCreators: [firstDummyFilter, secondDummyFilter],
+          config: configWithEmptyDefinedFetch,
+          getElemIdFunc: mockGetElemIdFunc,
+        })
+        const { elements, isPartial } = await adapter.fetch(mockFetchOpts)
+        expect(isPartial).toBeFalsy()
+        expect(elements).toHaveLength(getMetadataTypes().length)
+        const customObjectsQuery = (client.getCustomObjects as jest.Mock).mock.calls[0][1]
+        expect(customObjectsQuery.isTypeMatch('any kind of type')).toBeTruthy()
+        const fileCabinetQuery = (client.importFileCabinetContent as jest.Mock).mock.calls[0][0]
+        expect(fileCabinetQuery.isFileMatch('any/kind/of/path')).toBeTruthy()
+      })
+      it('with defined include without exclude, should fetch all types and instances in include', async () => {
+        const configWithIncludeButNoExclude = { ...configWithoutFetch,
+          [FETCH]: { [INCLUDE]: { types: [{ name: 'someType.*' }], fileCabinet: ['someFilePath'] } } }
```

Comment:
styling nit: (applies to many tests)
```suggestion
        const configWithIncludeButNoExclude = {
        ...configWithoutFetch,
        [FETCH]: {
          [INCLUDE]: {
            types: [{ name: 'someType.*' }],
            fileCabinet: ['someFilePath'],
          },
        },
      }
```